docs/dev/CodingGuidelines/CodingGuidelines.md

```diff
@@ -185,90 +185,85 @@ Overall Acuminator analyzers can be divided into two categories:
 
 #### Independent Analyzers
 
-Independent analyzers are analyzers which can't be easily grouped together because the checks they perform apply to code elements which have little in common. This makes any usefull code or data sharing impossible. 
-For example, one independent analyzer checks everywhere in code that the number of arguments passed to a BQL query corresponds to the number of parameters declared by the query. Another independent analyzer checks that everywhere in code 
-graphs are not instantiated with constructor. These two analyzers have nothing to share with each other.
+Independent analyzers are analyzers which cannot be easily grouped together. This happens because the checks performed by independant analyzers are applied to code elements that have little in common. This feature makes any usefull code or data sharing impossible. 
+For example, one independent analyzer checks everywhere in code that the number of arguments passed to a BQL query corresponds to the number of parameters declared by the query. Another independent analyzer checks everywhere in code that graphs are not instantiated with a constructor. These two analyzers have nothing to share with each other.
 
-Independent analyzers should be derived form a `PXDiagnosticAnalyzer` class. This class provides base support for Acuminator infrastructure, unit tests and some general checks. It also intializes and supplies to derived analyzers following data:
-* Acuminator code analysis settings. It is a collection of flags: is code analysis enabled, is ISV mode enabled and etc. See `CodeAnalysisSettings` class declaration for more details.
-* `PXContext` object. It is a wrapper over Roslyn `Compilation` which contains Roslyn `ISymbol` symbols specific to Acumatica. If you need to add a new symbol you should add it to `PXContext` or to one of its collection of symbols.  
+Independent analyzers should be derived form the `PXDiagnosticAnalyzer` class. This class provides base support for the Acuminator infrastructure, unit tests, and some general checks. It also intializes derived analyzers and supplies them with the following data:
```

Comment:
my mistype - "from" instead of "form"
```suggestion
Independent analyzers should be derived from the `PXDiagnosticAnalyzer` class. This class provides base support for the Acuminator infrastructure, unit tests, and some general checks. It also intializes derived analyzers and supplies them with the following data:
```